site/style.css

```
@@ -1,223 +1,577 @@
+  flex: 1;
+  padding: 2.5rem;
+}
+
+.ex-panel{
+  display: none;
+  animation: fadeIn 0.3s ease;
 }
-.ex-panel{display:none}
 .ex-panel.active{display:block}
+
+@keyframes fadeIn {
```

Stylelint (17.3.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)